src/public/app/shared/animations/router-transition.animation.ts

```diff
@@ -0,0 +1,34 @@
+import { group, trigger, animate, transition, style, query, animateChild } from '@angular/animations';
+
+export const routerTransition = trigger('routerTransition', [
+    transition('* => *', [
+        query(':enter, :leave', style({ position: 'absolute', top: 0, left: 0, right: 0 })),
+        query(':leave', style({ zIndex: 100 })),
+        query(':enter', style({ transform: 'translateY(100%)' })),
+
+        group([
+            query(':leave', group([
+                animate('500ms cubic-bezier(.35,0,.25,1)', style({ transform: 'translateY(-100%)' })), // y: '-100%'
+                animateChild()
+            ])),
+            query(':enter', group([
+                animate('500ms cubic-bezier(.35,0,.25,1)', style({ transform: 'translateY(0%)' })),
+                animateChild()
+            ]))
+        ])
+    ]),
+    // transition('* <=> *', [
```

Comment:
Can you remove commented code?

Reply:
I left it so that we could use it later. I just didn't feel like writing it down somewhere else but I can remove it if you want.